techniques/aria/ARIA23.html

```diff
@@ -30,24 +30,24 @@ <h3>Updating the contents of a chat conversation</h3>
 				<section class="description">
 					<p>Comments that users type into a chat input field are appended to the end of the chat history region. The region is marked with role of <code>log</code> so that new additions are announced by ATs. When each new chat message appears, a screen reader should announce its content (depending on AT/browser compatibility).</p>
 				</section>
-<pre xml:space="preserve"><code class="language-html">&lt;div id="chatRegion" role="log" aria-labelledby="chatHeading"&gt;
+<pre><code>&lt;div id="chatRegion" role="log" aria-labelledby="chatHeading"&gt;
   &lt;h4 id="chatHeading"&gt;Chat History&lt;/h4&gt;
   &lt;ul id="conversation"&gt;
     &lt;li&gt;The latest chat message&lt;/li&gt;
   &lt;/ul&gt;
 &lt;/div&gt;</code></pre>
-				<p class="working-example"><a href="../../working-examples/aria-role-log/chatlog.html">Working example: using <code class="language-html">role="log"</code> with chat conversation</a></p>
+				<p class="working-example"><a href="../../working-examples/aria-role-log/chatlog.html">Working example: using <code>role="log"</code> with chat conversation</a></p>
 			</section>
 			<section class="example">
 				<h3>Updating the log of a server</h3>
 				<p>An application log records time-stamped activities. The log is exposed in the app as a view, with the region marked with the role of <code>log</code> so that the new additions are announced by the ATs. (The default value for the <code>aria-relevant</code> attribute is "additions", so the removal of the old top entries due to log size limitations will not be announced.)  When each new log entry is added, a screen reader announces it.</p>
```

Comment:
```suggestion
				<p>An application log records time-stamped activities. The log is exposed in the app as a view, with the region marked with <code>role="log"</code> so that the new additions are announced by the ATs. (The default value for the <code>aria-relevant</code> attribute is "additions", so the removal of the old top entries due to log size limitations will not be announced.)  When each new log entry is added, a screen reader announces it.</p>
```

and then benefit from highlighting